packages/api-explorer/src/scenes/DiffScene/DocDiff/DocDiff.spec.tsx

```diff
@@ -109,4 +109,30 @@ describe('DocDiff', () => {
       ).toBeInTheDocument()
     })
   })
+
+  it('paginates with correct number of entries per page', async () => {
```

Comment:
is it worth checking the last row isn't the first row on the next page?

Reply:
Yes I agree and feel it wouldn't hurt. See the new additional test.